modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/display/context/CPSearchResultsDisplayContext.java

```diff
@@ -77,6 +78,11 @@ public CPSearchResultsDisplayContext(
 			ConfigurationProviderUtil.getPortletInstanceConfiguration(
 				CPSearchResultsPortletInstanceConfiguration.class,
 				_cpRequestHelper.getThemeDisplay());
+
```

Comment:
I don't think this empty line is needed. Can you please remove it and verify with the sf?